openrtb_ext/imp_cwire.go

```diff
@@ -0,0 +1,10 @@
+package openrtb_ext
+
+// ImpExtCwire defines the contract for MakeRequests `request.imp[i].ext.bidder`
+type ImpExtCWire struct {
```

Comment:
I missed this in my review yesterday but you don't need this at all. The imp ext struct is only needed for adapters that unmarshal the imp ext to retrieve the bidder params and do some conversions based off of that before sending a value to their bidder service. In your case looks like you support openrtb2 format and are sending the original marshalled request as is so you can delete this definition completely.

Reply:
It true that we don't need it currently, but it doesn't hurt to have it. If I remember correctly from the docs, it can be used to check if the parameters are malformed and if they are, we can ignore the impression (aka not make an HTTP call). We don't do that currently, but maybe we will. All in all, I personally think they also can be seen as a part of documentation just like `cwire.json`.